respirators-101.sublime-project

```diff
@@ -8,7 +8,7 @@
 			],
 			"folder_exclude_patterns":
 			[
-				"node_modules"
+//				"node_modules"
```

Comment:
Did you mean to commit this?  This line keeps all of the npm dependencies from showing up in/slowing down the Goto Anything dialog or in file searches, and keeps the duplicated-by-symlink files in `lib` and `data` from showing up there too.

Reply:
Yeah, I wasn't sure how to handle this.  For this project, I wanted to include it because there's a bunch of libraries in here that I'm unfamiliar with, so I like to Ctrl+Shift+F to search *everything*.  But that's more developer preference, and doesn't need to be committed.  But I wasn't comfortable adding the file to .gitignore either.

Reply:
I tend to use [npm-home](https://www.npmjs.com/package/npm-home) in the CLI, and then browse around the Github repo to read.  It's easier than diving into the `node_modules` morass.

Reply:
Ha, a potential solution would be to have a separate .sublime-project file.  `respirators-101-duff.sublime-project` and `respirators-101-botkin.sublime-project`

Reply:
Do you use the Goto Anything functionality (Ctrl+P)?  I was pushed to exclude `node_modules` just to keep it snappy.